.pullapprove.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+
+group_defaults:
+  always_pending:
+    labels:
+      - do-not-merge
+      - needs-rebase
+      - work-in-progress
+  approve_by_comment:
+    enabled: true
+    approve_regex: '^:+1:'
+  author_approval:
+    ignored: true
+
+groups:
+  develop-reviewers:
+    required: 1
```

Comment:
Were only requiring 1 reviewer to push to develop? I though it was 2 all the way through. I could be wrong though.